var/spack/repos/builtin/packages/wt/package.py

```diff
@@ -0,0 +1,68 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Wt(CMakePackage):
+    """Wt, C++ Web Toolkit
+
+    Wt is a C++ library for developing web applications."""
+
+    homepage = "http://www.webtoolkit.eu/wt"
+    url = "https://github.com/emweb/wt/archive/3.3.7.tar.gz"
+
+    version('3.3.7', '09858901f2dcf5c3d36a9237daba3e3f')
+    version('master', branch='master',
+            git='https://github.com/emweb/wt.git')
+
+    # wt builds in parallel, but requires more than 5 GByte RAM per -j <njob>
+    # which most machines do not provide and crash the build
+    parallel = False
+
+    variant('openssl', default=True,
+            description='SSL and WebSockets support in the built-in httpd, '
+                        'the HTTP(S) client, and additional cryptographic '
+                        'hashes in the authentication module')
+    variant('libharu', default=True, description='painting to PDF')
+    # variant('graphicsmagick', default=True,
+    #         description='painting to PNG, GIF')
+    variant('mariadb', default=False, description='MariaDB/MySQL backend')
+    variant('postgresql', default=False, description='PostgreSQL backend')
+    # variant('firebird', default=False, description='Firebird backend')
```

Comment:
those "backend" only build [database object (DBO) creation](https://github.com/emweb/wt/blob/3.3.7/CMakeLists.txt#L103-L105), which should not be mutually exclusive. But this is the test still open in the PR description.